app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,19 @@
     <string name="app_name">SampleApp</string>
     <string name="title_activity_main">MainActivity</string>
     <string name="title_activity_second">SecondActivity</string>
+
+    <!-- Map Screen -->
```

Comment:
Praise: Love the use of constants for text in the UI!

Reply:
Thanks I appreciate it 😄